config/core/ressources/mongodbsource.yaml

```diff
@@ -0,0 +1,139 @@
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Let's do one thing, to make it simpler, so that you don't have to specify all the fields in the yaml, because they get easily outdated. Follow this example:

https://github.com/knative-sandbox/sample-source/blob/master/config/300-samplesource.yaml

Reply:
Done